tests/ui/crashes/ice-6332.stderr

```diff
@@ -0,0 +1,22 @@
+error: immediately dereferencing a reference
+  --> $DIR/ice-6332.rs:3:5
+   |
+LL |     *&bad = true;
+   |     ^^^^^ help: try this: `bad`
+   |
+   = note: `-D clippy::deref-addrof` implied by `-D warnings`
+
+error[E0308]: mismatched types
+  --> $DIR/ice-6332.rs:3:13
+   |
+LL |     *&bad = true;
+   |             ^^^^ expected `&mut bool`, found `bool`
+   |
+help: consider dereferencing here to assign to the mutable borrowed piece of memory
+   |
+LL |     **&bad = true;
+   |     ^^^^^^
```

Comment:
There is a compiler error in the test. Is this reproducible without this error?

Reply:
Indeed, I guess I focused too much on the error message than the panic itself